gh-127805: Clarify Formatter initialization in logging.rst (#127850)
python/cpython: Doc/library/logging.rst

   .. method:: Handler.setFormatter(fmt)

>       Sets the :class:`Formatter` for this handler to *fmt*. The *fmt* argument must be an initialized :class:`Formatter` instance.

Reviewer: The argument can also be `None`.

>       Sets the :class:`Formatter` for this handler to *fmt*. The *fmt* argument must be an initialized :class:`Formatter` instance or ``None``.

Reviewer 2: ```suggestion
      Sets the :class:`Formatter` for this handler to *fmt*.
      The *fmt* argument must be a :class:`Formatter` instance or ``None``.
```

Just to avoid a too long line (80-chars max). Btw, what do you mean by "initialized" instance? Once it's created, the instance is ready for use right?

   .. method:: Handler.setFormatter(fmt)

>       Sets the :class:`Formatter` for this handler to *fmt*.

Reviewer 3: ```suggestion
      Sets the formatter for this handler to *fmt*.
```
